src/js/edu-benefits/components/EducationView.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { formatReviewDate } from '../../common/schemaform/helpers';
+
+export default function EducationView({ formData }) {
+  let from = '';
+  let to = '';
+  if (formData.dateRange) {
+    from = formatReviewDate(formData.dateRange.from);
+    to = formatReviewDate(formData.dateRange.to);
+  }
+
+  return (
+    <div>
```

Comment:
@goldenmeanie Do we generally want to display all of the entered data here, or just a summary to identify the item?
Screenshot:
![image](https://cloud.githubusercontent.com/assets/25439097/23776977/674e9694-04e6-11e7-9a63-43720c58a408.png)